unistd/arpa/nameser.h

```diff
@@ -0,0 +1,57 @@
+// arpa/nameser.h
+// Provided by brimdata on 2022/10/19
```

Comment:
Have you talked to the Brim guys about whether they're ok with providing their implementations upstream?